javalee/src/main/java/javalee/com/ManageStationsController.java

```diff
@@ -0,0 +1,113 @@
+package javalee.com;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.layout.StackPane;
+import javalee.com.bd_connection.DbConnection;
+import javalee.com.entities.Cities;
+import javalee.com.entities.City;
+import javalee.com.entities.Station;
+import javalee.com.entities.Stations;
+
+public class ManageStationsController {
+    @FXML
+    private TextField identificador;
+
+    @FXML
+    private ComboBox<String> typeMeasurementCity;
+    
+    @FXML
+    private ComboBox<String> typeMeasurementStation;
+
+    @FXML
+    ObservableList<String> rowsTypeMeasurementStation;
+
+    @FXML
+    private void initialize(){
+        typeMeasurementStation.setDisable(true);
+        identificador.setDisable(true);
+        Cities cities = new Cities();
+        List<City> listaDeCidades = cities.getAllCity();
```

Comment:
Por algum motivo, não se é problema no meu, mas eu tomo erro de:
`Caused by: java.lang.Error: Unresolved compilation problem: 
        The method getAllCity() is undefined for the type Cities`